sources/packages/backend/apps/db-migrations/src/sql/Student/Add-col-pd-status.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS pd_status sims.permanent_disability_status;
```

Comment:
I hope that it will not open a can of worms about naming possibilities 😉 
I believe that the acronyms for PD/PPD mean
- PPD: Persistent or Prolonged Disability
- PD: Permanent Disability
With the above in mind I believe that instead of `permanent_disability_status` it should be only `disability_status`.

Reply:
As per discussion with the dev team, we decided to use disability status instead of permanent disability status and try to change the existing code as much as possible.